shared/common-adapters/text.desktop.js

```diff
@@ -203,6 +203,19 @@ export const styles = {
     lineHeight: '19px',
     letterSpacing: '0.3px'
   },
+  textBodyBig: {
```

Comment:
if this is for the input (under special styles) it appears as 29px line height to me in zeplin. also i think if these are used in these special cases we should maybe put them somewhere else in this file like some 'special' export or something